### DISCUSSION TITLE
Is this the correct thread for GitHub Desktop questions?

### DISCUSSION THREAD

**Discussion by kunkel321** [community/community#118621] [New to GitHub]
I'm a total noob to GitHub and GitHub Desktop.   If it matters, I'm not a professional programmer--just a hobbyist.  I have a couple of AutoHotkey apps that I've written with VSCode.  Some folks on the AHK forums have suggested that I put my stuff on GitHub.  I have some noob questions, but first I want to make sure that this is the correct location....  

Edit:  Sorry... Meant to ask, "Is this the correct SUB FORUM, not thread.

**Comment by Luuan11** [comment]
No worries! It sounds like you're on the right track. GitHub is a popular platform for hosting and sharing code, and GitHub Desktop is a user-friendly tool for managing your GitHub repositories. As a hobbyist programmer, it's a great idea to share your AutoHotkey apps on GitHub, especially if you're seeking feedback or collaboration.

As for your question about the correct "subforum," GitHub organizes repositories into "repositories" rather than "subforums." Each repository is like a folder for your project where you can store code, documentation, and other files. So, yes, GitHub is indeed the correct platform for sharing your projects.

Feel free to ask your noob questions about GitHub or GitHub Desktop hehe

good week!

**Comment by kunkel321** [comment]
Thank you for the reply, @Luuan11.  I won't keep this off-topic discussion going too long, but I'm trying to learn to use this forum, so I must ask:  Where is your reply?   I was able to read it, because of the notification email that arrived.   But now, I'm back and the github discussion, and I don't see it anywhere...  Was it removed?   
EDIT:  Sorry!  I see it now.  It is "folded."  I have to click to unfold you reply.  Then I can read it.  Thanks Again.  :)  
EDIT Again... Strange, I cannot seem to mark your (Luan's) response as "Mark as answer."
